src/Apps/Order/Routes/Shipping2/ShippingContext.tsx

```diff
@@ -148,9 +149,18 @@ export const ShippingContextProvider: FC<{
     setStage: (payload: ShippingStage) => {
       dispatch({ type: "SET_STAGE", payload })
     },
+  }
 
-    handleExchangeError,
+  const goBackToFulfillmentDetails = () => {
+    if (state.stage !== "fulfillment_details") {
+      actions.setStage("fulfillment_details")
+    }
+  }
```

Comment:
I put this action here thinking we may want it to grow in time (for example to un-select the current shipping quote) but this might not be necessary.